src/app/lib/views/lang_dropdown.js

```diff
@@ -1,47 +1,80 @@
 (function (App){
     'use strict';
 
+    var objectAssign = require('es6-object-assign').assign;
```

Comment:
There is no need to defined twice objectAssign (if defined in global, it's already accessible)